src/config.rs

```diff
@@ -15,10 +15,17 @@ fn default_timeout_ms() -> u16 {
     250
 }
 
+fn default_command() -> Vec<String> {
+    Vec::new()
+}
+
 #[derive(Clone, Debug, Deserialize)]
 pub struct CornerConfig {
     pub output: Option<OutputConfig>,
-    pub command: Vec<String>,
+    #[serde(default = "default_command")]
+    pub enter_command: Vec<String>,
```

Comment:
```suggestion
    #[serde(alias = "command"]
    pub enter_command: Vec<String>,
```
This will make it backwards compatible.

Additionally, could it be made so either `enter_command` (or `command`) and `exit_command` are required? It should error instead of doing nothing when no command is set.